tests/unit/test_deletions_query_generator.py

```diff
@@ -0,0 +1,73 @@
+import shutil
+from pathlib import Path
+
+from dqgen.adapters.resource_fetcher import get_file_content
+from dqgen.services.deletions_query_generator import InstanceDeletionsGenerator, SimplePropertyDeletionsGenerator, \
+    ReifiedPropertyDeletionsGenerator
+
+
+def test_instance_deletions_generator():
```

Comment:
a test of this kind would be sufficient to cover all the generators, if you decide to move the logic into the parent class